modules/administration-guide/partials/proc_collecting-dev-workspace-engine-metrics-with-prometheus.adoc

```diff
@@ -0,0 +1,62 @@
+[id="proc_collecting-dev-workspace-engine-metrics-with-prometheus_{context}"]
+= Collecting {devworkspace} engine metrics with Prometheus
+
+[role="_abstract"]
+This section describes how to use the Prometheus to collect, store, and query metrics about the {devworkspace} engine.
+
+.Prerequisites
+
+* The `devworkspace-controller-metrics` service is exposing metrics on port `8443`.
+
+* The `devworkspace-webhookserver` service is exposing metrics on port `9443`.
+
+* Prometheus 2.26.0 or later is running and the Prometheus console is running on port `9090` with a corresponding *service* and *route*. See link:https://prometheus.io/docs/introduction/first_steps/[First steps with Prometheus].
+
```

Comment:
would be nice to somehow introduce the section which describes the usage of the OpenShift in-cluster Prometheus for OCP. Can be done as a separate PR